src/accelerate/commands/config/cluster.py

```diff
@@ -198,13 +198,24 @@ def get_cluster_input():
                 )
 
                 deepspeed_devices = ["none", "cpu", "nvme"]
+                deepspeed_offload_nvme_path = ["/nvme", "/local_nvme"]
                 if deepspeed_config["zero_stage"] >= 2:
                     deepspeed_config["offload_optimizer_device"] = _ask_options(
                         "Where to offload optimizer states?", deepspeed_devices, lambda x: deepspeed_devices[int(x)]
                     )
                     deepspeed_config["offload_param_device"] = _ask_options(
                         "Where to offload parameters?", deepspeed_devices, lambda x: deepspeed_devices[int(x)]
                     )
+                    deepspeed_config["offload_param_nvme_path"] = _ask_options(
```

Comment:
This should only be asked if the offload device is nvme (as it won't be useful otherwise).